examples/experimental/horizontal-pod-autoscaling/README.md

```diff
@@ -0,0 +1,217 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/experimental/horizontal-pod-autoscaling/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Horizontal Pod Autoscaler
+
+Horizontal pod autoscaling is an experimental feature in Kubernetes 1.1.
+It allows the number of pods in a replication controller or deployment to scale automatically based on observed CPU or memory usage.
+<b>Please note that the current API is tentative and will be subject to change before a stable version is released.</b>
+
+In this document we explain how this feature works by walking you through an example of enabling horizontal pod autoscaling with the php-apache server.
+
+## Prerequisites
+
+This example requires a running Kubernetes cluster in the version 1.1 with experimental API enabled on API server (``--runtime-config=experimental/v1alpha1=true``),
+and experimental controllers turned on in controller manager (``--enable-experimental=true``).
```

Comment:
I believe this is changing due to #15427. In general, it might be better to replace this paragraph with a pointer to the "API groups" section that Eric is adding in #15339, so you don't have to keep trying to keep it up-to-date.